docs/docs/examples-tutorials/recipes/running-tracetest-with-opensearch.md

```diff
@@ -132,6 +132,7 @@ services:
 
   tracetest:
     image: kubeshop/tracetest:v0.8.0
```

Comment:
Does it make sense to update the version here?

```suggestion
    image: kubeshop/tracetest:v0.9.0
```

Reply:
Yeah, I see a couple of examples with a fixed tracetest version. We can create a new ticket for that

Reply:
Applied to all pinned versions